addons/microsoft_calendar/models/calendar.py

```diff
@@ -521,6 +521,9 @@ def _microsoft_values(self, fields_to_sync, initial_values={}):
         if 'location' in fields_to_sync:
             values['location'] = {'displayName': self.location or ''}
 
+        if 'videocall_location' in fields_to_sync:
```

Comment:
Thanks for your work!
It seems we need more checks here like we have done in `google_calendar`   will it require if location is set before generating a outlook link. because what is see if event with location will not need outlook link.

Also make sure remove video call URL for time off event. As those are not required an videocall link.
here is reference for it: https://github.com/odoo/odoo/pull/162205

Reply:
Hello @kpt-odoo,

Thank you for your feedback! I have now implemented the necessary checks for location settings before generating an Outlook link, as suggested. Additionally, I have removed the video call URL for time-off events.

Thanks again for your suggestions! 😊